### PR TITLE
feat: Allow image to be passed to main component

### LINKED ISSUE #88: Allow an image to be passed to the main component
Rather than the current hardcoded zebrafish or uploading, we should be able to pass in imageData somehow when initialising the package

### PR DESCRIPTION
**Pull request by SilviaZeta**
# Description

Added `slicesData` as a prop to `UserInterface`, put file loading code in `examples/src/index.tsx` and `examples/src/autoload.ts` to autoload zebrafish-heart.jpg and pass it in as an `ImageBitmap[][]`. Removed legacy image loading code from `BackgroundCanvas`, now it's much simpler and just shows whatever it's passed as `props.displayedImage`. Also simplified the minimap - now there's just a single minimap component that draws the white rectangle on top of another `BackgroundCanvas`, and handles the mouse events.

This PR also fixes:
- Brightness and contrast are not persistent when resizing;
- Minimap does not render at first load and appears/disappears when resizing.

closes #88, #101

# Dependency changes
No

# Testing
N/A

# Documentation
N/A

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] ~I have made corresponding changes to the documentation~
- [x] My changes generate no new warnings
- [ ] ~I have added tests that prove my fix is effective or that my feature works~
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by cooper667 — APPROVED** [review]
Such an easy minimap fix if you know what you're doing 😁

**Comment by SilviaZeta** [comment]
> Such an easy minimap fix if you know what you're doing 😁

yes, shamefully easy! Looks like encapsulation can result in some odd, entertaining bugs. :-)

File: src/baseCanvas/Minimap.tsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { PositionAndSize } from "@/baseCanvas/Canvas";
 import { Props as BaseProps, BaseCanvas } from "./Canvas";
 
 export interface Props extends BaseProps {
+  imageWidth: number;
```
Comment (ChasNelson1990) [inline src/baseCanvas/Minimap.tsx:7]: Do we ever use one of these without the other? Should they not just be a single prop holding both numbers?

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]

File: src/baseCanvas/Minimap.tsx (hunk 1 above)
Comment (SilviaZeta) [reply src/baseCanvas/Minimap.tsx]: turns out imageWidth and imageHeight weren't needed. thanks!

**Review by cooper667 — APPROVED** [review]

**Review by ChasNelson1990 — APPROVED** [review]

**Pull request merged**